examples/basic/hello-world.ts

```diff
@@ -41,27 +41,13 @@ import { pinoPrettyLogger } from "@golem-sdk/pino-logger";
       logger,
     });
 
-    const workload = Package.create({
-      imageTag: demand.demand.image,
+    const allocation = await glm.payment.createAllocation({ budget: 1 });
+    const demandSpecification = await glm.market.buildDemand(demand.demand, allocation);
```

Comment:
I don't think we will need allocation at all here, there was an idea that decorations from allocation could be generated manually based on the requestor id..